packages/main/src/main.spec.ts

```diff
@@ -172,3 +180,60 @@ describe('ElectronApp#on window-all-closed', () => {
     expect(ELECTRON_APP_MOCK.quit).toHaveBeenCalledOnce();
   });
 });
+
+describe('AppPlugin', () => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const plugins: Array<new (...args: any[]) => AppPlugin> = [DefaultProtocolClient];
```

Comment:
can we avoid any there ?

Reply:
found a work-around by using the prototype, not really a big fan, bug avoid using the any